packages/mercurius/README.md

```diff
@@ -29,7 +29,7 @@
 > **NOTE** `mercurius` is not compatible with Express.
 
 ```bash
-$ npm i --save @nestjs/graphql @nestjs/mercurius mercurius graphql
```

Comment:
could we bring them ($) back? following what we have in other repositories

Reply:
I think so, it is my editor who has removed them by default  😄

Reply:
great, thank you!

Reply:
fix done, see here: [7ae6fd8](https://github.com/nestjs/graphql/pull/3367/commits/7ae6fd84b41ca21c98f9dc0400e5d29b1c3f9fa9)